Client/TabManagement/TabMigrationUtility.swift

```diff
@@ -126,6 +126,7 @@ class DefaultTabMigrationUtility: TabMigrationUtility {
         prefs.setBool(false, forKey: migrationKey)
         // Ecosia: Tabs architecture implementation from ~v112 to ~116
         clearDeprecatedArchive()
+        Analytics.shared.migration(true)
```

Comment:
If I read the code correctly, I think this line will run in both cases success AND error, won't it? Thus a count on this event will reflect the number of overall migrations. That just needs to be considered when looking at the data later.